data_processing/wdl/workflows/hifi_qc_workflow.wdl

```diff
@@ -2,50 +2,137 @@ version 1.0
 
 import "../tasks/read_stats.wdl" as read_stats_wf
 import "../tasks/ntsm.wdl" as ntsm_check
+import "../tasks/methylationCheck.wdl" as meth_check  
 
 workflow hifi_qc_wf {
     input {
-        Array[File] hifi_reads
+        File hifi_reads
         Array[File] other_reads
         String sample_id
+        Boolean perform_methylation_check = false  # Optional input to perform methylation check
     }
 
     parameter_meta {
         hifi_reads: "Reads to QC. Can be fastq, fastq.gz, or bam."
         other_reads: "Trusted reads to compare against to check for sample swaps. Can be fastq, fastq.gz, or bam."
         sample_id: "Id to use for output naming."
+        perform_methylation_check: "Set to true to perform methylation check on BAM files."
     }
 
     meta {
-        author: "Julian Lucas"
-        email: "juklucas@ucsc.edu"
+        author: "Julian Lucas, Andrew Blair"
```

Comment:
I think you can just switch this to you. I think of this field as "who should I email with questions"